src/utilities/pcall.py

```
@@ -0,0 +1,53 @@
+
+        await pcall(succeed)              # (123, True)
+        await pcall(fail())               # (ValueError(...), False)
+        await pcall(fail, (ValueError,))  # (ValueError(...), False)
+    """
+    try:
+        if callable(coro) and inspect.iscoroutinefunction(coro):
+            coro = coro()
+
+        result = await (coro() if callable(coro) else coro)
+        return result, True
+    except handle if handle else (Exception,) as e:
```

Ruff (0.12.2)

`except` handlers should only be exception classes or tuples of exception classes

(B030)